### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by pavanjoshi914**
Fixes: #609 
add installation section to readme
add link for conventional commits specification without version

### CODE REVIEW

**Comment by pavanjoshi914** [comment]
@bumi @escapedcat

**Review by escapedcat — COMMENTED** [review]
lgtm

**Comment by escapedcat** [comment]
nit: According to conventional-commits the commit-message should start with `docs`, i.e. `docs: update readme`

**Comment by escapedcat** [comment]
No worries. No need to repeat `readme` as a `type` as well I think. It's already mentioned in the subject.  

It's a bit like writing i.e. `fix(button): fix the button click-handler`. Better remove duplication and focus on the main thing i.e. `fix(button): click-handler`.  

But all good. Not too important for now really.

**Comment by pavanjoshi914** [comment]
> No worries. No need to repeat `readme` as a `type` as well I think. It's already mentioned in the subject.
> 
> It's a bit like writing i.e. `fix(button): fix the button click-handler`. Better remove duplication and focus on the main thing i.e. `fix(button): click-handler`.
> 
> But all good. Not too important for now really.

good point!  changed

**Comment by pavanjoshi914** [comment]
i think its more readable now

File: README.md [hunk 1]
```diff
@@ -42,10 +42,19 @@ Download Nightly releases
...
-- [Chrome](https://chrome.google.com/webstore/detail/alby/iokeahhehimjnekafflcihljlcjccdbe) (Opera, Brave, and all Chromium based browsers)
-- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/alby/)
+Alby supports
+
+- All [Chromium based browsers](https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium) - Chrome Opera, Brave etc.
+- Firefox
 - more coming soon...
 
+## Installation
+
```
Comment (bumi) [inline README.md:52]: now that we have this section. 
can we move the `## Try out the most recent version` to here. 

So under `## Installation` people find the links to the latest official releases in the stores and to the nightly builds.

Comment (pavanjoshi914) [reply README.md:52]: can we say official releases as stable versions? and nightly releases as a most recent alpha version

Comment (bumi) [reply README.md:52]: yep

Comment (pavanjoshi914) [reply README.md:43]: done

**Review by bumi — APPROVED** [review]
great. thanks!

**Pull request merged**